module/VuFind/src/VuFind/ILS/Driver/AlephFactory.php

```diff
@@ -66,7 +66,10 @@ public function __invoke(ContainerInterface $container, $requestedName,
         return parent::__invoke(
             $container,
             $requestedName,
-            [$container->get(\VuFind\Cache\Manager::class)]
+            [
+                $container->get(\VuFind\Cache\Manager::class),
+                $container->get('Laminas\Mvc\I18n\Translator')
```

Comment:
I would suggest using `\Laminas\Mvc\I18n\Translator::class` here instead of the string; this will allow static analysis to behave more effectively.